package.json

```diff
@@ -5,9 +5,6 @@
   "homepage": "https://fribbels.github.io/hsr-optimizer",
   "dependencies": {
     "@ant-design/icons": "^5.2.6",
-    "@testing-library/jest-dom": "^5.17.0",
```

Comment:
i'm hoping OOTB webpack treeshaking didn't ship testing harness code with the prod bundles...